docs/docs/feature-library/activity-skill-filter.md

```diff
@@ -2,62 +2,144 @@
+## Overview
 
-- they can be ordered
-- visibility of activities can be filtered by worker skills
+This feature allows administrators to configure visibility of Agent Activities based on skills, as well as ordering the Activities in the Activities drop down menu.
 
-## Known issue
+---
 
-On older Twilio Flex accounts a TaskRouter Activity of "Reserved" may be present. This Activity is automatically hidden by the Flex UI and breaks the logic of the plugin. In this scenario it is advised you remove any TaskQueue references to the "Reserved" activity (which will likely be the activity to use when task is assigned on the TaskQueue and as such isn't actually used by Flex) and then delete the "Reserved" Activity.
+## Business Details
+
+### context
+It is not uncommon to create Activities that should only ever be used or set programmatically.  
 
-## Disclaimer
+It might also be the case that only certain users should be able to set particular Activities.  
 
-The Flex Component "Activity" which is the activity selector and current status indicator at the top right of the flex screen, lacks appropriate properties or hooks to modify the OOTB behavior without completely replacing the component.
+Although minor it is also a common request for agents to see the most frequently used Activities first in the drop down menu.
 
-This plugin uses a wrapper around the Activity component to apply CSS to the underlying component to order and hide the activity elements. This is not an ideal approach but is a tidy solution until appropriate hooks become available. An alternative approach would be to completely rewrite the underlying components but this would be a lot more invasive and magnitudes of effort greater.
+### objective
+This feature aims to address these needs by providing the administrator the ability to lock down Activities by skill as well as ordering them based on a set order then alphabetically.
 
-## Configuration
+## Known Issues
 
-This plugin relies on custom configuration being applied to your underlying [Flex configuration](https://www.twilio.com/docs/flex/developer/ui/configuration#modifying-configuration-for-flextwiliocom). This is accomplished using the [Flex Configuration Updater](https://github.com/twilio-professional-services/twilio-proserv-flex-project-template/tree/main/flex-config) package in this repository.
+### issue one
+On older Twilio Flex accounts a TaskRouter Activity of "Reserved" may be present. This Activity is automatically hidden by the Flex UI and breaks the logic of the plugin. In this scenario it is advised you remove any TaskQueue references to the "Reserved" activity (which will likely be the activity to use when task is assigned on the TaskQueue and as such isn't actually used by Flex) and then delete the "Reserved" Activity.
 
-In your `ui_attributes.{environment}.json` file, enable the "activity_skill_filter" feature and add a rule for each activity SID you want to configure:
+## Flex User Experience
 
-```
+N/A
```

Comment:
We should have something here. Maybe a "before/after" screenshot showing an activities menu with system activities, and another without them and reordered?